src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -27,15 +27,15 @@
         <% if(request.getSession().getAttribute("user") != null){ %>
           <h1>New Conversation</h1>
           <form action="/conversations" method="POST">
-              <div class="form-group">
+              <div class="form-group" style="margin-bottom: 16px;">
                 <label class="form-control-label">Title:</label>
               <input type="text" name="conversationTitle">
             </div>
 
             <button type="submit">Create</button>
           </form>
 
-          <hr/>
+          <hr class="groove"/>
```

Comment:
Again, the styling for the `groove` class is applied in the `main.css` stylesheet